flocker/cli/script.py

```diff
@@ -54,10 +56,15 @@ class DeployOptions(Options):
     """
 
     synopsis = ("Usage: flocker-deploy [OPTIONS] "
-                "DEPLOYMENT_CONFIGURATION_PATH APPLICATION_CONFIGURATION_PATH"
+                "CONTROL_HOST DEPLOYMENT_CONFIGURATION_PATH "
+                "APPLICATION_CONFIGURATION_PATH"
```

Comment:
:question: 

The "all upper case letters for required arguments" idiom here was weird before and still is.  `<control host> ...` would be more conventional, I think.